src/main/java/nextstep/subway/auth/application/AuthService.java

```diff
@@ -27,7 +27,10 @@ public TokenResponse login(TokenRequest request) {
         return new TokenResponse(token);
     }
 
-    public LoginMember findMemberByToken(String credentials) {
+    public LoginMember findMemberByToken(String credentials, boolean skipException) {
+        if (isSkipRequest(credentials, skipException)) {
+            return new LoginMember();
```

Comment:
```suggestion
            return LoginMember.empty();
```
기본 생성자로 만들면 값들이 null로 초기화될테니 팩토리 메서드를 만들고 값을 초기화하는 방법도 있습니다

Reply:
여기서 팩토리 메서드를 만나는군요 👍 

좋은 것 같습니다 !

꼼꼼한 피드백 감사해요 현구님 :)